app/main.ipynb

```diff
@@ -100,18 +127,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "float_array = np.array([1.2, 5.9, 4.2, 8.9, 7.6, 5.6, 1.1])\n",
+    "float_array[[1, 3, -1]]"
```

Comment:
The task requires retrieving the first, third, and last elements of the array. The current indexing retrieves the second, fourth, and last elements. Use indices [0, 2, -1] to get the correct elements.